suitesparse.sh

```diff
@@ -0,0 +1,13 @@
+
+if [[ $1 =~ refs/tags/v([0-9]\.[0-9]\.[0-9]).*$ ]];
+then
+    VERSION=${BASH_REMATCH[1]}
+else
+    exit -1
+fi
+
+curl -L https://github.com/DrTimothyAldenDavis/GraphBLAS/archive/refs/tags/v${VERSION}.tar.gz | tar xzf -
+cd GraphBLAS-${VERSION}/build
+cmake ..
+make -j$(nproc)
+make install
```

Comment:
Small nits:
- should we be explicit and specify `-DCMAKE_BUILD_TYPE=Release`?  IIRC, it used to default to Debug, but I think it now defaults to Release.
- any value in building with more verbosity, `VERBOSE=1`?

Reply:
> Small nits:
> 
>     * should we be explicit and specify `-DCMAKE_BUILD_TYPE=Release`?  IIRC, it used to default to Debug, but I think it now defaults to Release.

Why not!  Added.  I'm no cmake expert but it does look to default to Debug.
> 
>     * any value in building with more verbosity, `VERBOSE=1`?

On cmake?  Not sure what this produces, if it's useful to you I'll add it.